.dockerignore

```diff
@@ -1,6 +1,11 @@
+.git
 *.pyc
 .idea
-.git
+.coverage
 .tox
+.cache
+dist/
+build/
 docs/
```

Comment:
Seems pretty unlikely... but perhaps you might want access to the docs in a scenario where you're debugging something on-site, with lewis installed only as a docker image, and on an isolated network that isn't connected to the internet.

It's a pretty specific / unlikely scenario, of course, but on the other hand I don't see much downside to including the contents of /docs... plus it has the upside of keeping the two ignore files more consistent with each other.